site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,159 +3,76 @@ layout: 'layouts/doc-post.njk'
-
-<table class="fixed-table width-full">
-   <tbody>
-      <tr>
-         <th style="margin-left:0; padding-left:0">Invalid pattern</th>
-         <th style="margin-left:0; padding-left:0">Why it's invalid</th>
-      </tr>
-      <tr>
-         <td><code>https://www.google.com</code></td>
-         <td>No <em>path</em></td>
-      </tr>
-      <tr>
-         <td><code>https://*foo/bar</code></td>
-         <td>'*' in the <em>host</em> can be followed only by a '.' or '/'</td>
-      </tr>
-      <tr>
-         <td><code>https://foo.*.bar/baz&nbsp;</code></td>
-         <td>If '*' is in the <em>host</em>, it must be the first character</td>
-      </tr>
-      <tr>
-         <td><code>http:/bar</code></td>
-         <td>Missing <em>scheme</em> separator ("/" should be "//")</td>
-      </tr>
-      <tr>
-         <td><code>foo://*</code></td>
-         <td>Invalid <em>scheme</em></td>
-      </tr>
-      <tr>
-         <td><code>chrome://*</code></td>
-         <td>Unsupported <em>scheme</em></td>
-      </tr>
-      <tr>
-         <td><code>chrome-extension://*</code></td>
-         <td>Unsupported <em>scheme</em></td>
-      </tr>
-      <tr>
-         <td><code>about:*</code></td>
-         <td>Unsupported <em>scheme</em>.</td>
-      </tr>
-   </tbody>
-</table>
-
-{% Aside 'success' %}
-Content scripts can be injected into related frames like `about:`, `data:`, etc by setting `"match_origin_as_fallback"` to `"true"` in the manifest. See [Injecting in related frames][cs-frames] for details.
-{% endAside %}
+: Chrome doesn't support match patterns for [top Level domains (TLD)][mdn-tld]. Specify your match patterns within individual TLDs, as in `http://google.es/*` and `http://google.fr/*`.
+
+## Example patterns {: #examples }
+
+`https://*/*` or `https://*/`
+: Matches any URL using the `https` scheme.
+
+`https://*/foo*`
+: Matches any URL using the `https` scheme, on any host, with a path that starts with `foo`. Examples of matches include `https://example.com/foo/bar.html` and `https://www.google.com/foo`.
+
+`https://*.google.com/foo*bar`
+: Matches any URL using the `https` scheme, on a google.com host, with a path that starts with `foo` and ends with `bar`. Examples of matches include `https://www.google.com/foo/baz/bar` and `https://docs.google.com/foobar`.
+
+`file:///foo*`
+: Matches any local file whose path starts with `foo`. Examples of matches include `file:///foo/bar.html` and `file:///foo`.
+
+`http://127.0.0.1/*` or `http://127.0.0.1/`
+: Matches any URL that uses the `http` scheme and is on the host 127.0.0.1. Examples of matches include `http://127.0.0.1/` and `http://127.0.0.1/foo/bar.html`.
+
+`http://localhost/*`
+: Matches any localhost port.
+
+`*://mail.google.com/` or `*://mail.google.com/*`
+: Matches any URL that starts with `http://mail.google.com` or `https://mail.google.com`.
+
+`urn:*`
+: Matches any URL that starts with `urn:`. Examples of matches include `rn:uuid:54723bea-c94e-480e-80c8-a69846c3f582`.
```

Comment:
We should drop this if it doesn't work (it doesn't as far as I can tell from testing).

Reply:
Removed.